blog/good-compliance-programs-do-not-require-fancy-platforms.md

```diff
@@ -0,0 +1,235 @@
+
+**The report**
+
+The auditor then issues a report.
+In their report they include their assessment of the controls and evidence, as well as a "system description" of the organization that describes the company and its products in a fairly high degree of detail.
+
+The report that the auditor issues is subsequently, more often than not, an extremely comprehensive analysis and honest view of the organization and how it operates.
+
+**Conclusion**
+
+**Therefore, in recent years, a compliance report (such as SOC2) has become a cornerstone factor in whether a customer decides to place their trust in the organization.
+In a lot of cases, many businesses won't even consider a B2B relationship if the other business is not compliant within a framework.**
+
+That is why you should care.
+Because if your organization is compliant within a framework, then it is following best practices to the best of its abilities, and thus your customers can feel good about placing their trust with you.
+
+</details>
+
+So what role do these platforms play in helping you do that?
+
+Well, each one has a slightly different angle.
+Almost all of them promote themselves as a way to "simplify" the compliance process by providing a set of tools to help you organize your resources and automate compliance processes, particularly around an audit.
+Personally, I think you should be wary of any of these platforms that say they can make things "simple" for you, because that is silly.
+This stuff is complex.
+What I do value, are the platforms that can help you understand how your internal operations map to compliance framework criteria.
+That is the art, the heart, and the best kept secret of this whole thing that is so often glazed over.
+
+Consider this well known maxim:
+
+> Give a man a fish, feed him for a day. Teach a man to fish, feed him for a lifetime.
+
+Most of these platforms teach only when they have to, as a means to the end of getting you to pay to use their tools.
+They don't want to teach, they want you to keep coming back to them.
+The truth is, once you learn how to parse compliance framework criteria language and learn how to map real-world operations to it, you don't need these platforms.
+
+Now, to be fair, learning this is not super easy.
+
+I recently asked a SOC2 expert I know, [Martin Cozzi](https://www.marana.io/), how he learned and he said it was the same way I was learning: through experience (he has been through 50+ audits).
+There is no course that he knows of that teaches this.
+
+And yet, the more you learn, the more it becomes apparent, at least to me, that what makes a compliance audit easy, versus a slogging and demoralizing haze, is not whether you have fancy automation tools, a fancy web ui, or an all-in-one auditing platform, its whether your company has embraced the framework against which you wish to be compliant and understands how your operations fit into it.
```

Comment:
This sentence starts with the word And + there are 8 commas. Only two of those are used to list things.